### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by NurkaAmre**
### Learn how to use **API** in js
- Created new game using **API**
- Implemented `"Refresh"` button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
- Implement the form `"Submit"` button that will save a score for a game created by you (sending data to the API).
- Used a`rrow function` & `await `& `async`

### CODE REVIEW

File: src/modules/post.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    body: JSON.stringify(
+      formData,
+    ),
+  })
+    .then((response) => response.json())
+    .then((data) => data.result);
+//     .then((response) => response.json())
+//     .then((data) => console.log(data));
+//   // .then((data) => data.result);
```
Comment (christianbiring1) [inline src/modules/post.js]: - [ ] Kindly make sure you delete the commented code as they make your code base looks ugly.

File: src/index.js [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 });
 
-leaderForm.addEventListener('submit', () => {
-  const leader = new LeaderBoard(nameInput.value, scoreInput.value);
-  nameInput.value = '';
-  scoreInput.value = '';
-  leaderArr.push(leader);
-  displayList(leaderArr);
+refreshBtn.addEventListener('click', () => {
+  displayList();
```
Comment (christianbiring1) [inline src/index.js]: - [ ] Kindly make sure the table of the score is permanently visible on the page for a better UX. It should not be visible only when the refresh button is hit and disappear when the page refresh.
`Hint: You may consider using the getValue f(x) here in the refresh click event and make the table static to see the records.`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @NurkaAmre , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptuve PR `title` and `summary`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (NurkaAmre) [reply src/index.js]: Thank you review and suggestions.

**Review by christianbiring1 — APPROVED** [review]
Hi @NurkaAmre , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Implement request changes :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NurkaAmre** [comment]
Thank you 😎 .

**Pull request merged**